### PR TITLE
Add prettier code formatter

### PR DESCRIPTION
**Pull request by jamessspanggg**
For more info on the options that Prettier provides, can look here: [Prettier Options](https://prettier.io/docs/en/options.html).

### CODE REVIEW

**Comment by jamessspanggg** [comment]
@GiftForGood/core-developers let me know if the configurations are okay for y'all.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "export": "next build && next export"
+    "export": "next build && next export",
+    "lint": "prettier --check pages/*.js",
```
Comment (jamessspanggg) [inline package.json:21]: the check location can be changed in the future if we have other js files that we need to check

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "bracketSpacing": true,
+    "jsxBracketSameLine": false,
+    "trailingComma": "es5",
+    "useTabs": true,
+    "tabWidth": 2,
+    "semi": true,
+    "printWidth": 80,
```
Comment (jinyingtan) [inline .prettierrc.json:8]: Should we go with 120 instead? @kohchihao @DanKhoo

Comment (kohchihao) [reply .prettierrc.json:8]: Lets change to `120`. haha `80` feels too short!

File: package.json (hunk 1 above)
Comment (kohchihao) [reply package.json:21]: noted.

File: .prettierrc.json (hunk 2 above)
Comment (jamessspanggg) [reply .prettierrc.json]: done.

**Pull request merged**